### DISCUSSION TITLE
关于音质问题

### DISCUSSION THREAD

**Discussion by Ni-Cmyk** [1715173329/UnblockNeteaseMusic#807] [Q&A]
ENABLE_FLAC=true  MIN_BR=320000 -e 这个2个我设置了为么还是播放的标准音质呢，但是有的歌曲又是高品质   切换下一曲的时候又变回了标准音质， 必须手动切换才行呢

**Comment by 1715173329** [comment]
当您开启此选项时，我们会向对应音源请求无损音质下载链接，但如果该平台没有无损音源，或因收费等原因，我们也无能为力。
